protocol.md

```diff
@@ -1056,6 +1094,11 @@ export interface TextDocumentSyncOptions {
 
 interface ServerCapabilities {
 	/**
+	 * The server supports receiving results solely through $/partialResult notifications for requests from the server to the client.
+	 * If true, the client is allowed to let the result in the final response be null.
```

Comment:
`If true, the server should send all data in $partialResults and the final response should be null.`